API.md

```diff
@@ -241,4 +247,8 @@ Copy a file to the node via SSH as the given user.
 
 #### `ip_address`
 
-The IP address of the node.
+The external IP address of the node.
+
+#### `private_ip_address`
+
+The DC/OS internal IP address of the node if different from the external IP address.
```

Comment:
Could you add a little bit of information here as to what this is used for. Also, what do you think of having this required, rather than `None` ever - I guess it can be the same as `ip_address` where appropriate.